modular_skyrat/modules/customization/modules/clothing/~donator/donator_items.dm

```diff
@@ -0,0 +1,22 @@
+//Donator reward for Otome
+//give it to them in the loadout or something similar, if you ever remove these cigs from the vending machine
+/obj/item/clothing/mask/cigarette/khi
+	name = "\improper Kitsuhana Singularity cigarette"
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/masks.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/mask.dmi'
+	icon_state = "khioff"
+	icon_on = "khion"
+	icon_off = "khioff"
+	type_butt = /obj/item/cigbutt
+	list_reagents = list(/datum/reagent/drug/nicotine = 10, /datum/reagent/toxin/mindbreaker = 5)
```

Comment:
Mindbreaker cigs are still a contraband drug, as evidenced by the fact there are already mindbreaker cigs in the cig vendor contraband.